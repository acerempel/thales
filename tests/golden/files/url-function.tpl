{url(load-template("url-function/index.html"))}
{url(load-template("url-function/potato.html"))}
{url(load-template("url-function/artichoke/index.html"))}
{url(load-template("url-function/styles.css"))}
{url(load-yaml("url-function/styles.css"))}
