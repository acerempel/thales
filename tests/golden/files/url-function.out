url-function
url-function/potato
url-function/artichoke
url-function/styles.css
url-function/styles.css
